### ISSUE TITLE
Renpy can't figure out it's installed the Android SDK

### ISSUE THREAD

**Issue by Jordaldo** [renpy/renpy#1276]
I downloaded renpy / rapt / android sdk / java sdk, took me a sec to figure out that I needed to personally set the java.home thing, but I can't seem to find anything similar that I would have to do for the android sdk, it just won't work.

I've downloaded renpy, let it install everything. No dice.
I've downloaded the beta version, let it install everything. No dice.
I've unzipped the folders and put them in renpy myself (Trying to figure out what people were talking about here). No dice.
The android SDK now doesn't come with the sdk manager that someone was talking about, I had to download this whole android studio shenanigans to get access to the sdk manager, and I still can't figure out what it is that renpy needs from those files if it is even the right thing.

TLDR: Renpy installation doesn't work with Android SDK.

**Comment by renpytom** [comment]
I believe this was a transient bug, as discussed in #1284. It looks like it's fixed. 

You shouldn't need to set java.home or anything like that, as long as you have a 32-bit Java installed Ren'Py should do the rest for you.
